packages/transaction-controller/src/TransactionController.test.ts

```diff
@@ -686,19 +801,24 @@ describe('TransactionController', () => {
           },
         ];
 
+        // this feels like a weird thing to test here
         const pendingTransactions =
           nonceTrackerMock.mock.calls[0][0].getPendingTransactions(
             ACCOUNT_MOCK,
           );
 
-        expect(nonceTrackerMock).toHaveBeenCalledTimes(1);
+        // gets called in constructor now
+        expect(nonceTrackerMock).toHaveBeenCalledTimes(4);
```

Comment:
should we comment where these 4 times come from?

Reply:
working on this

Reply:
https://github.com/MetaMask/core/pull/3643/commits/81fcd605873eb74e6f05d47cb9f85a46a4d2d216